packages/eventual-send/src/index.d.ts

```diff
@@ -26,9 +44,6 @@
 
 export type Callable = (...args: any[]) => any;
 
-// Same as https://github.com/microsoft/TypeScript/issues/31394
-export type ERef<T> = PromiseLike<T> | T;
-
 export declare const EmptyObj: {};
```

Comment:
Internal-only.

Reply:
There are no more references to `EmptyObj` and unless I am missing something, I suppose this was substituted with `{}`.

@michaelfig Do you want me to check or just drop it?

Reply:
Drop, please.